config/config.local.sample.neon

```diff
@@ -41,6 +41,10 @@ parameters:
 		key: "AKIAIZGO5LP3LHAUFQRQ"
 		secret: # https://my.1password.com/vaults/all/allitems/lb4q5qjwxfqqaqf5f7jzxr3z7a
 
+	imgproxy:
+		key: "0aa4c34cb6636fb8d4deacd150b1c7b4"
+		salt: "20030cf6ef8fc43168d77a7c05f4cd31"
```

Comment:
To by asi v public repozitáři být nemělo.

Reply:
omg to jsem si nevšimnul, to mě nevyhovuje že je tohle public repo :( diky zes to chytnul

Reply:
Tyhle údaje jsou fake?

Reply:
ne, ale moc se mi to nechce řešit, nevidim duvod proc je to repo public :( muzu to dat do 1 pwd, ale to pak vede na to ze to stejne akorat vsichni kopiruju z jinych projektu a nepouziva se aktualni verze

Reply:
jestli dobře chápu, tak při založení projektu, by to mohl doplnit ten PR, co se automaticky vytvoří... ale jako.

No public je, protože to třeba někdy někoho zaujme a bude u nás chtít pracovat :D jako třeba vojta přišel kvůli mango-cli...

Reply:
ok kazdopadne tohle je za me low prio, ted to mergnu a pak to vyresime kdyztak casem, protoze to je zmena jenom v tomhle repu a ne v kazdem projektu co to pouziva